braddeibert/musclenerds
# Track workouts!!!

An alarming mix of both Java hodgepodge and wizardry in this one. Beware of reading the code. 🌶️  

CHANGES:
* WorkoutTracking activity now gathers data from each set of every exercise in the workout, and stores in the db upon completion
* 🐛   infested. Be nice when testing.

TESTING:
* Build app and run. Open database inspector in Android Studio and open the TRACKEDWORKOUT and TRACKEDSET. These tables should both be empty.
* Track any workout of your choice. Enter weight, reps, and use difficulty slider. Then, click "Track Set".
* When at the end of the workout, click "Workout complete!" to submit data.
* Check the database inspector again. TRACKEDWORKOUT and TRACKEDSET should contain your new tracked data!

KNOWN BUGS:
* Too many to count at this point, probably.
